library/src/main/java/com/sloydev/preferator/Preferator.kt

```diff
@@ -0,0 +1,10 @@
+package com.sloydev.preferator
+
+import android.content.Context
+import android.content.Intent
+
+object Preferator {
+  fun launch(context: Context) {
```

Comment:
This function (and the no-op version) should be annotated with `@JvmStatic` to keep Java compatibility.

Reply:
added!